### PR TITLE
Update Angular to v17

### PR DESCRIPTION
**Pull request by GoodDayForSurf**

### CODE REVIEW

File: apps/demos/Demos/ActionSheet/Basics/Angular/app/app.component.ts [hunk 1]
```diff
@@ -29,9 +29,7 @@ export class AppComponent {
 
 @NgModule({
   imports: [
-    BrowserModule,
-    BrowserTransferStateModule,
-    DxActionSheetModule,
+    BrowserModule,    DxActionSheetModule,
```
Comment (dxvladislavvolkov) [inline apps/demos/Demos/ActionSheet/Basics/Angular/app/app.component.ts]: Move DxActionSheetModule to new line

File: apps/demos/Demos/ActionSheet/PopoverMode/Angular/app/app.component.ts [hunk 2]
```diff
@@ -46,9 +46,7 @@ export class AppComponent {
 
 @NgModule({
   imports: [
-    BrowserModule,
-    BrowserTransferStateModule,
-    DxActionSheetModule,
+    BrowserModule,    DxActionSheetModule,
```
Comment (dxvladislavvolkov) [inline apps/demos/Demos/ActionSheet/PopoverMode/Angular/app/app.component.ts]: Move DxActionSheetModule to new line

File: apps/demos/Demos/Autocomplete/Overview/Angular/app/app.component.ts [hunk 3]
```diff
@@ -92,9 +92,7 @@ export class AppComponent {
 
 @NgModule({
   imports: [
-    BrowserModule,
-    BrowserTransferStateModule,
-    DxAutocompleteModule,
+    BrowserModule,    DxAutocompleteModule,
```
Comment (dxvladislavvolkov) [inline apps/demos/Demos/Autocomplete/Overview/Angular/app/app.component.ts]: Move  DxAutocompleteModule to new line

File: apps/demos/Demos/ButtonGroup/Overview/Angular/app/app.component.ts [hunk 4]
```diff
@@ -35,9 +35,7 @@ export class AppComponent {
 
 @NgModule({
   imports: [
-    BrowserModule,
-    BrowserTransferStateModule,
-    DxButtonGroupModule,
+    BrowserModule,    DxButtonGroupModule,
```
Comment (dxvladislavvolkov) [inline apps/demos/Demos/ButtonGroup/Overview/Angular/app/app.component.ts]: Move to new line. You should fix it in all demos

File: apps/demos/Demos/HtmlEditor/Mentions/Angular/app/app.component.html [hunk 5]
```diff
@@ -56,9 +56,9 @@
     <span
       class="dx-mention"
       spellcheck="false"
-      data-marker="@"
+      data-marker="&#64;"
```
Comment (dxvladislavvolkov) [inline apps/demos/Demos/HtmlEditor/Mentions/Angular/app/app.component.html:59]: Why did you change it?

**Review by ivanblinov2k17 — DISMISSED** [review]

File: apps/demos/Demos/HtmlEditor/Mentions/Angular/app/app.component.html (hunk 5 above)
Comment (GoodDayForSurf) [reply apps/demos/Demos/HtmlEditor/Mentions/Angular/app/app.component.html:59]: angular throws error here if use @

File: packages/devextreme-angular/package.json [hunk 6]
```diff
@@ -18,21 +18,22 @@
   "author": "Developer Express Inc.",
   "license": "MIT",
   "peerDependencies": {
-    "@angular/common": ">12.0.0",
-    "@angular/core": ">12.0.0",
-    "@angular/forms": ">12.0.0",
+    "@angular/common": ">15.0.0",
```
Comment (dxvladislavvolkov) [inline packages/devextreme-angular/package.json]: Change to >=16.0.0

**Review by dxvladislavvolkov — DISMISSED** [review]

**Review by ivanblinov2k17 — APPROVED** [review]

**Review by alexslavr — APPROVED** [review]
